src/main/java/test.java

```diff
@@ -0,0 +1,11 @@
+import browser.Driver;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+
+public class test {
+    @Test
+    public void startDriver(){
+        Driver.getDriver().get(Driver.getProperty("login.url"));
```

Comment:
I don't think that you receive some property configurations from Driver. Try to separate Driver and Property reader